rest-service/manager_rest/resources.py

```diff
@@ -432,9 +443,7 @@ def get(self, blueprint_id, _include=None):
         """
         Get blueprint by id
         """
-        blueprint = get_blueprints_manager().get_blueprint(blueprint_id,
-                                                           _include)
-        return responses.BlueprintState(**blueprint.to_dict())
+        return get_blueprints_manager().get_blueprint(blueprint_id, _include)
```

Comment:
have only models returned from endpoint methods
wrap models with responses in marshal_with, dont pass resource_fields but rather the response class
have response classes be aware of how to convert model into response